src/providers/github/Github.ts

```diff
@@ -214,6 +217,50 @@ export class Github {
         e.message = `Error creating release: ${e.message}`;
         throw e;
       });
+
+    return release.data.id;
+  }
+
+  async uploadReleaseAssets({
+    releaseId,
+    assetsDir,
+    ignorePattern,
+    fileNamePrefix
+  }: {
+    releaseId: number;
+    assetsDir: string;
+    ignorePattern?: RegExp;
+    fileNamePrefix?: string;
+  }) {
+    for (const file of fs.readdirSync(assetsDir)) {
+      // Used to ignore duplicated legacy .tar.xz image
+      if (ignorePattern && ignorePattern.test(file)) continue;
+
+      const filepath = path.resolve(assetsDir, file);
+      const contentType = mime.lookup(filepath) || "application/octet-stream";
+      try {
+        // The uploadReleaseAssetApi fails sometimes, retry 3 times
+        await retry(
+          async () => {
+            await this.octokit.repos.uploadReleaseAsset({
+              owner: this.owner,
+              repo: this.repo,
+              release_id: releaseId,
+              data: fs.createReadStream(filepath) as any,
+              headers: {
+                "content-type": contentType,
+                "content-length": fs.statSync(filepath).size
+              },
+              name: `${fileNamePrefix || ""}${path.basename(filepath)}`
```

Comment:
Adds prefix for multivariant packages. For example:

If variant is holesky for Obol package, `content-hash` file name will be `holesky-obol_content-hash`